application/messages/messages_test.go

```diff
@@ -53,3 +64,135 @@ func validateEmails(ts *TestSuite, td testData, testEmailer notifications.DummyE
 		ts.Contains(msgs[i].Subject, w, "incorrect email subject")
 	}
 }
+
+func validateNotificationUsers(ts *TestSuite, tx *pop.Connection, td testDataNew) {
+	var notnUsers models.NotificationUsers
+	ts.NoError(tx.Where("email_address in (?)",
+		td.wantToEmails[0], td.wantToEmails[1]).All(&notnUsers))
+
+	ts.Equal(len(td.wantToEmails), len(notnUsers), "incorrect count of NotificationUsers")
+	for _, n := range notnUsers {
+		n.Load(tx)
+		notn := n.Notification
+		ts.Contains(notn.Subject, td.wantSubjectContains, "incorrect notification subject")
+		ts.Contains(notn.InappText, td.wantInappTextContains, "incorrect notification inapp text")
+
+		for _, c := range td.wantBodyContains {
+			ts.Contains(notn.Body, c, "incorrect notification body")
+		}
+
+		ts.WithinDuration(time.Now().UTC(), n.SendAfterUTC, time.Minute,
+			"incorrect NotificationUser SendAfterUTC")
+	}
+}
+
+func (ts *TestSuite) Test_SendQueuedNotifications() {
+	t := ts.T()
+	db := ts.DB
+
+	f := getClaimFixtures(db)
+
+	user := f.Policies[0].Members[0]
+
+	// Create queued notifications for different scenarios
+	//  (already sent, should be sent, not ready to be sent)
+
+	type notnFixture struct {
+		name      string
+		sendAfter time.Time
+		sentAt    nulls.Time
+		notnUser  models.NotificationUser
+	}
+
+	// The database truncates the time in the fractions of seconds somehow
+	alreadySentTime := time.Now().UTC().Add(-domain.DurationWeek).Truncate(time.Second)
```

Comment:
```suggestion
	alreadySentTime := time.Now().UTC().Add(-1 * domain.DurationWeek).Truncate(time.Second)
```
To make the negative more visible

Reply:
:+1: